src/Prelude.purs

```diff
@@ -314,9 +270,16 @@ liftA1 f a = pure f <*> a
 -- |    y <- m2 x
 -- |    m3 x y
 -- | ```
-class (Apply m) <= Bind m where
+class Apply m <= Bind m where
   bind :: forall a b. m a -> (a -> m b) -> m b
 
+infixl 1 bind as >>=
+
+bindFlipped :: forall m a b. Bind m => (a -> m b) -> m a -> m b
```

Comment:
Oops, missed that comment, I'll add it.

Yep, although with `bind` rather than `extend` :wink: